githubapp/client_creator.go

```diff
@@ -208,10 +213,10 @@ func (c *clientCreator) newClient(base *http.Client, details string) (*github.Cl
 	return client, nil
 }
 
-func (c *clientCreator) newV4Client(base *http.Client, details string) (*githubv4.Client, error) {
+func (c *clientCreator) newV4Client(base *http.Client, details string, installID int64) (*githubv4.Client, error) {
 	ua := makeUserAgent(c.userAgent, details)
 
-	middleware := append([]ClientMiddleware{setUserAgentHeader(ua)}, c.middleware...)
+	middleware := append([]ClientMiddleware{setUserAgentHeader(ua), setInstallationIDHeader(installID)}, c.middleware...)
```

Comment:
Not that it breaks anything, but do you know if the rate limit headers are used for the v4 API? I seem to recall that you have to ask for rate limiting information as part of the query instead.

Reply:
Good point, it is different: https://developer.github.com/v4/guides/resource-limitations/#returning-a-calls-rate-limit-status